exporter/signalfxexporter/internal/translation/constants.go

```diff
@@ -55,258 +55,260 @@ translation_rules:
     user: true
     wait: true
 - action: aggregate_metric
-  metric_name: system.cpu.usage
+  metric_name: sf_temp.system.cpu.usage
   aggregation_method: sum
   without_dimensions:
   - state
 - action: copy_metrics
   mapping:
-    system.cpu.delta: system.cpu.total
+    sf_temp.system.cpu.delta: sf_temp.system.cpu.total
 - action: aggregate_metric
-  metric_name: system.cpu.total
+  metric_name: sf_temp.system.cpu.total
   aggregation_method: sum
   without_dimensions:
   - state
 - action: calculate_new_metric
   metric_name: cpu.utilization_per_core
-  operand1_metric: system.cpu.usage
-  operand2_metric: system.cpu.total
+  operand1_metric: sf_temp.system.cpu.usage
+  operand2_metric: sf_temp.system.cpu.total
   operator: /
 - action: copy_metrics
   mapping:
-    cpu.utilization_per_core: cpu.utilization
+    cpu.utilization_per_core: sf_temp.cpu.utilization
 - action: aggregate_metric
-  metric_name: cpu.utilization
+  metric_name: sf_temp.cpu.utilization
   aggregation_method: avg
   without_dimensions:
   - cpu
+- action: multiply_float
+  scale_factors_float:
+    sf_temp.cpu.utilization: 100
 
```

Comment:
@rmfitzpatrick @dmitryax Hi, I've noticed that you're multiplying by 100 only `cpu.utilization` metric while `cpu.utilization_per_core` are kept as is.
This breaks the compatibility with existing charts & dashboards as Signalfx Smart Agent reported both metrics as percentages.